code/index.html

```diff
@@ -1,22 +1,136 @@
+                            leo nec, pharetra gravida dolor. Duis nec posuere mauris. In odio purus, vulputate sed massa sit amet, commodo vulputate velit.
+                            <br>
+                            <br> Donec mauris risus, consequat non est vehicula, dapibus dignissim ligula. Aliquam interdum ante vitae arcu aliquam, at imperdiet lacus pharetra. Mauris id accumsan ante. Interdum et malesuada fames ac ante ipsum primis
+                            in faucibus. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum sollicitudin sapien a tempor scelerisque. Donec vitae sollicitudin lectus, ut ultricies lectus. Donec pretium
+                            laoreet dapibus. In odio purus, vulputate sed massa sit amet, commodo vulputate velit.
+                            <br>
+                            <br> Etiam egestas, dui ac ultricies vestibulum, orci ex hendrerit nunc, vel dictum lectus mi ac velit. Quisque sed cursus arcu, quis ultricies sapien. Ut interdum sapien scelerisque augue semper efficitur. Donec mauris risus,
+                            consequat non est vehicula, dapibus dignissim ligula. Aliquam interdum ante vitae arcu aliquam, at imperdiet lacus pharetra. Mauris id accumsan ante. Interdum et malesuada fames ac ante ipsum primis in faucibus. Pellentesque
+                            habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.</p>
+                    </div>
+                </div>
+            </div>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <!-- The Smaller News  -->
+    <section class="container-smaller-news">
+
+        <div class="news-card">
+            <div class="greenwashing">
+                <img src="./Greenwashing.jpg" alt="How to spot greenwashing">
+                <div class="heading-greenwashing">
+                    <h2><span style="font-family: lucida calligraphy; font-size: 20px; text-decoration: underline; font-style: italic; color: rgba(30, 29, 31, 0.842);">Environmentally friendly?</span> How to spot greenwashing and make sure your garments really
+                        are sustainable</h2>
+                </div>
+            </div>
+        </div>
+
+        <div class="news-card">
+            <div class="billie-holiday">
+                <img src="./Prada-Billie-Holiday.jpg" alt="Prada to do costume for Bille Holiday movie">
+                <div class="heading-billie">
+                    <h2>Iconic fashion house <span style="font-family: lucida calligraphy; font-size: 20px; text-decoration: underline; font-style: italic; color: rgba(30, 29, 31, 0.842);">Prada</span> to make costumes for Billie Holiday in upcoming biopic</h2>
+                </div>
+            </div>
+        </div>
```

Comment:
Nice and clean cards! Looks like they all have the same styling though 🤔 Just give all cards the same class name in that case and style that one class :)